docs/source/configuration/metrics.mdx

```diff
@@ -56,20 +56,42 @@ apollo_router_http_request_duration_seconds_bucket{le="0.9"} 1
 
 > Note that if you haven't run a query against the router yet, you'll see a blank page because no metrics have been generated!
 
-### Currently available metrics
-The following metrics are available using Prometheus:
-
-- HTTP router request duration: `apollo_router_http_request_duration_seconds_bucket`
-- HTTP request duration by subgraph: `apollo_router_http_request_duration_seconds_bucket` with attribute `subgraph`
-- Total number of HTTP requests by HTTP Status: `apollo_router_http_requests_total`
-- Number of currently connected clients: `apollo_router_session_count_total`
-- Number of in flight GraphQL requests: `apollo_router_session_count_active`
-- Number of cache hits for different `kind` of cache (`apq`, `query planner`, `introspection`) and for different `storage` (`memory`, `redis`): `apollo_router_cache_hit_count`
-- Number of cache misses for different `kind` of cache (`apq`, `query planner`, `introspection`) and for different `storage` (`memory`, `redis`): `apollo_router_cache_miss_count`
-- Time to hit the cache for different `kind` of cache (`apq`, `query planner`, `introspection`) and for different `storage` (`memory`, `redis`), in seconds: `apollo_router_cache_hit_time`
-- Time to miss the cache for different `kind` of cache (`apq`, `query planner`, `introspection`) and for different `storage` (`memory`, `redis`), in seconds: `apollo_router_cache_miss_time`
-- Time spent processing a request, outside of waiting for external or subgraph requests, in seconds (`apollo_router_processing_time`)
-- Number of triggered timeouts: `apollo_router_timeout`
+### Available metrics
+The following metrics are available when using Prometheus. Where applicable, attributes are indicated.
+
+#### Http
+- `apollo_router_http_request_duration_seconds_bucket` - HTTP router request duration
+- `apollo_router_http_request_duration_seconds_bucket` - HTTP subgraph request duration, attributes:
+  - `subgraph`: (Optional) the subgraph being queried.
+- `apollo_router_http_requests_total` - Total number of HTTP requests by HTTP Status 
+- `apollo_router_timeout` - Number of triggered timeouts: 
```

Comment:
```suggestion
- `apollo_router_timeout` - Number of triggered timeouts
```

Reply:
Was there originally a list below this item that prompted the colon? Just making sure some content didn't get deleted by accident

Reply:
No there were no extra attributes